rpc_state_reader/src/lib.rs

```diff
@@ -530,6 +530,7 @@ mod tests {
         rpc_state.get_transaction(tx_hash);
     }
 
+    #[ignore]
```

Comment:
Why is this ignored? Can't the test be updated? If not, could you add a comment explaining why it's ignored?